tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/thirdparty/orc/WriterImpl.java

```diff
@@ -0,0 +1,2251 @@
+    }
+
+    /**
+     * Add a new value to the column.
+     * @param datum
+     * @throws IOException
+     */
+    void write(Datum datum) throws IOException {
+      if (datum != null && datum.isNotNull()) {
+        indexStatistics.increment();
+      } else {
+        indexStatistics.setNull();
+      }
+      if (isPresent != null) {
+        if(datum == null || datum.isNull()) {
+          foundNulls = true;
+          isPresent.write(0);
+        }
+        else {
+          isPresent.write(1);
+        }
+      }
+    }
+
+    void write(Tuple tuple) throws IOException {
+      if (tuple != null) {
+        indexStatistics.increment();
+      } else {
+        indexStatistics.setNull();
+      }
+      if (isPresent != null) {
+        if (tuple == null) {
+          foundNulls = true;
+          isPresent.write(0);
+        } else {
+          isPresent.write(1);
+        }
+      }
+    }
+
+    private void removeIsPresentPositions() {
+      for(int i=0; i < rowIndex.getEntryCount(); ++i) {
+        RowIndexEntry.Builder entry = rowIndex.getEntryBuilder(i);
+        List<Long> positions = entry.getPositionsList();
+        // bit streams use 3 positions if uncompressed, 4 if compressed
+        positions = positions.subList(isCompressed ? 4 : 3, positions.size());
+        entry.clearPositions();
+        entry.addAllPositions(positions);
+      }
+    }
+
+    /**
+     * Write the stripe out to the file.
+     * @param builder the stripe footer that contains the information about the
+     *                layout of the stripe. The TreeWriter is required to update
+     *                the footer with its information.
+     * @param requiredIndexEntries the number of index entries that are
+     *                             required. this is to check to make sure the
+     *                             row index is well formed.
+     * @throws IOException
+     */
+    void writeStripe(OrcProto.StripeFooter.Builder builder,
+                     int requiredIndexEntries) throws IOException {
+      if (isPresent != null) {
+        isPresent.flush();
+
+        // if no nulls are found in a stream, then suppress the stream
+        if(!foundNulls) {
+          isPresentOutStream.suppress();
+          // since isPresent bitstream is suppressed, update the index to
+          // remove the positions of the isPresent stream
+          if (rowIndexStream != null) {
+            removeIsPresentPositions();
+          }
+        }
+      }
+
+      // merge stripe-level column statistics to file statistics and write it to
+      // stripe statistics
+      OrcProto.StripeStatistics.Builder stripeStatsBuilder = OrcProto.StripeStatistics.newBuilder();
+      writeStripeStatistics(stripeStatsBuilder, this);
+      stripeStatsBuilders.add(stripeStatsBuilder);
+
+      // reset the flag for next stripe
+      foundNulls = false;
+
+      builder.addColumns(getEncoding());
+      if (streamFactory.hasWriterTimeZone()) {
```

Comment:
This timezone policy may be different from those of Tajo. Could you check the timezone? The following codes would be helpful to this work.

https://github.com/apache/tajo/blob/master/tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/text/TextFieldSerializerDeserializer.java#L56